### PR TITLE
THOR-720 - New typography styling

### PR DESCRIPTION
**Pull request by rartiss55**
Changes to font size/width throughout application to make things more consistent.

### CODE REVIEW

**Review by gitnordt — COMMENTED** [review]
I noticed that there is no icon showing up in the header like it is shown in the mockup. AIDA has an icon and it would be nice to put it inline with the title of the dashboard. Is this something that we can implement in a future task? Also, the font size in the taxonomy and network graph edge labels are larger than the standard 14px.
typographyfonts_thor-720

**Comment by Konnoke** [comment]
Just a note there's a difference in header style.
image

**Review by Konnoke — DISMISSED** [review]
Looks good! Love the new font

**Comment by rartiss55** [comment]
I added a couple updates from earlier PR comments -- please hold off on merging for now (waiting for UX approval on network graph related changes). 

**Update** - everything fine now on UX's end for network graph. Ready to be reviewed/merged.

File: src/app/components/aggregation/aggregation.component.scss
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/aggregation/aggregation.component.scss]: The `subheader` style is repeated in multiple widgets.  Could it be moved to the `BaseNeonComponent` SCSS?

File: src/app/components/about-neon/about-neon.component.scss
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/about-neon/about-neon.component.scss]: The `subheader` style here (in about-neon) is different than the `subheader` style in the widgets (the font size).  Should it be called something else to differentiate it?

**Comment by ThomasSchellenbergNextCentury** [comment]
@rartiss55 In `src/app/neon-properties.scss`, `vis-title-expected-height` should be reduced from `45px` to `42px` because the `mat-toolbar` border was reduced by 3.

**Comment by ThomasSchellenbergNextCentury** [comment]
@rartiss55 The change of header styles to be smaller and less-bold makes it harder for me to immediately differentiate between the header and non-header text.  It doesn't "pop" the same way.  Is it just me?  It that a silly concern?

File: src/app/components/aggregation/aggregation.component.scss
Comment (rartiss55) [reply src/app/components/aggregation/aggregation.component.scss]: Moved this as requested.

File: src/app/components/about-neon/about-neon.component.scss
Comment (rartiss55) [reply src/app/components/about-neon/about-neon.component.scss]: Renamed to about-subheader.

**Comment by rartiss55** [comment]
@ThomasSchellenbergNextCentury in response to your earlier comments:

- Updated the vis-title-expected-height.
- I think the header text is fine since it is still bolded, but we can talk to UX about this if you'd like.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Pull request merged**
